bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/coffeemachine.xml

```diff
@@ -28,6 +28,16 @@
 				<description>The identifier identifies one certain appliance on the ZigBee network.</description>
 				<required>true</required>
 			</parameter>
+			<parameter name="protocol" type="text">
+				<label>Protocol</label>
+				<description>The protocol used to communicate with the appliance</description>
+				<required>false</required>
+				<default>"ZigBee"</default>
+				<options>
+					<option value="LAN">LAN</option>
```

Comment:
Wouldn't actually be WLAN (instead of LAN) the better option as this is what people think of their Miele devices to speak?

Reply:
The option value is used internally in the binding (without modification), and in the protocol the value "LAN" is used

Reply:
Question still is whether protocol internals need to be exposed to the user. For the user, his device either speaks WLAN or ZigBee, this is what he knows, so he'd imho clearly expect those options to be offered here.